05-Mining-Protocol.md

```diff
@@ -72,11 +72,11 @@ The other portion of the block header that’s used to define the full search sp
 - Merkle root, deterministically computed from:
   - Coinbase transaction: typically 4-8 bytes, possibly much more.
   - Transaction set: practically unbounded space.
-    All roles in Stratum v2 MUST NOT use transaction selection/ordering for additional hash space extension.
-    This stems both from the concept that miners/pools should be able to choose their transaction set freely without any interference with the protocol, and also to enable future protocol modifications to Bitcoin.
-    In other words, any rules imposed on transaction selection/ordering by miners not described in the rest of this document may result in invalid work/blocks.
+    All roles in Stratum v2 must not use transaction selection for additional hash space extension.
+    This stems both from the concept that pools should be able to choose their transaction set freely without any interference with the protocol, and also to enable future protocol modifications to Bitcoin.
+    In other words, any rules imposed on transaction selection by miners not described in the rest of this document may result in invalid work.
 
-Mining servers MUST assign a unique subset of the search space to each connection/channel (and therefore each mining device) frequently and rapidly enough so that the mining devices are not running out of search space.
+Mining servers must assign a unique subset of the search space to each channel (and therefore each mining device) frequently and rapidly enough so that the mining devices are not running out of search space.
```

Comment:
I'm not sure we should remove these. My understanding is that a connection is just a simpler way of explaining a channel.